client/settings/digital-wallets/index.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	Card,
+	CardBody,
+	CardDivider,
+	CheckboxControl,
+} from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+const DigitalWallets = () => {
+	const [ isEnabled, setIsEnabled ] = useState( false );
+	const [ sectionsStatus, setSectionsStatus ] = useState( {
+		checkout: true,
+		productPage: true,
+		cart: true,
+	} );
+
+	const makeHandleSectionStatusChange = ( section ) => ( status ) => {
+		setSectionsStatus( ( oldStatuses ) => ( {
+			...oldStatuses,
+			[ section ]: status,
+		} ) );
+	};
+
+	return (
+		<Card className="digital-wallets">
+			<CardBody size="large">
+				<CheckboxControl
+					checked={ isEnabled }
+					onChange={ setIsEnabled }
+					label={ __(
+						'Enable digital wallets & express payment methods',
+						'woocommerce-payments'
+					) }
+					help={ interpolateComponents( {
+						mixedString: __(
+							// eslint-disable-next-line max-len
+							"By enabling this feature, you agree to {{stripeLink}}Stripe{{/stripeLink}}, {{appleLink}}Apple{{/appleLink}}, {{googleLink}}Google{{/googleLink}} and {{microsoftLink}}Microsoft{{/microsoftLink}}'s terms of use.",
```

Comment:
Do you mean split in multiple lines so it's also visually displayed to the merchant in multiple lines? Or just within the code?
I noticed that in other places the string is kept as it is going to be displayed - otherwise I'm afraid that it won't be pulled correctly to translators 👀

Reply:
Oh, I didn't know that! TIL! 👍